pkgs/development/compilers/rust/make-rust-platform.nix

```diff
@@ -18,9 +18,11 @@ rec {
     inherit cargo;
   };
 
+  fetchCargoVendor = buildPackages.callPackage ../../../build-support/rust/fetch-cargo-vendor.nix { inherit cargo; };
```

Comment:
fetchCargoVendor sounds about right because it's is fetching the crate deps as well as git deps.

`fetchCargoGitDeps` could work but again, we are fetching the crate deps as well.

Reply:
I think `fetchCargoVendor` good enough.